pkgs/applications/networking/browsers/vivaldi/default.nix

```diff
@@ -87,6 +88,7 @@ in stdenv.mkDerivation rec {
     done
     wrapProgram "$out/bin/vivaldi" \
       --add-flags ${lib.escapeShellArg commandLineArgs} \
+      --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--enable-features=UseOzonePlatform --ozone-platform=wayland}}" \
```

Comment:
ozone is enabled by default starting with 93.0.4530.5 so this flag isn't needed
https://www.reddit.com/r/swaywm/comments/nswd6w/ozonex11_is_now_the_default_in_chrome_unstable/

it could also be removed from other chromium-based program wrappers.